attributes/network/11_tokens/cddl/1_visual_logo.cddl

```diff
@@ -0,0 +1,45 @@
+; A single logo that can be shown for representing the token. A token can
```

Comment:
The parent folder should probably be named `11_ledger_tokens` to match the attribute name.

Reply:
Done, but we haven't been very strict about it so far. Would probably be useful to refactor the others in a separate PR.